tagsets/query.php

```diff
@@ -876,9 +877,9 @@ function query__apply_permanent_queries() {
                     $additional_clauses=array($active_clause,$exptype_clause,$notyetassigned_clause);
                     $query=query__get_query($query_array,time(),$additional_clauses,'');
                     $result=or_query($query['query'],$query['pars']);
-                    $p_is_eligibe=false;
-                    while ($pc=pdo_fetch_assoc($result)) if ($pc['participant_id']=$p['participant_id']) $p_is_eligibe=true;
-                    if (!$p_is_eligibe) $continue=false;
+                    $p_is_eligible=false;
+                    while ($pc=pdo_fetch_assoc($result)) if ($pc['participant_id']==$p['participant_id']) $p_is_eligible=true;
+                    if (!$p_is_eligible) $continue=false;
```

Comment:
I think it is possible to use `continue` keyword instead of variable to skip to next iteration (and get rid of using $continue within foreach completely)

Reply:
For the moment, I would rather keep the $continue logic. First, I think I use loop "continue" keywords nowhere int he code. Second, I cannot use it for the outer loop over participants (or_participants table needs to be updated below for anyone).

Reply:
That said, I fixed the loop syntax to use curly brackets.